utils/add-attribution.sh

```diff
@@ -13,11 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -o errexit -o nounset -o pipefail
 
-BLUEPRINT_NAME=$1
-BLUEPRINT_VERSION=$2
+PKG_PATH="${1:-.}"
+BLUEPRINT_NAME="${BLUEPRINT_NAME:-$(grep -m 1 "name:" "${PKG_PATH}/Kptfile" | sed 's/[[:space:]]*name:[[:space:]]*//')}"
```

Comment:
I'd rather have this be an input argument, or at least optionally one.

Reply:
it is an optional argument. you can supply via env var:
```
BLUEPRINT_NAME="blah" utils/add-attribution.sh <path>
```

Reply:
Could we allow `utils/add-attribution.sh <path>  <blueprint-name>`?

Reply:
The path is optional and the blueprint name is optional. Usually it's undesirable to have more than one optional argument. I would have to implement flags/options, and the libraries for it are tediously platform specific in bash...

Reply:
I guess, I don't love this new approach of trying to implicitly grab values but it's fine.

Reply:
Why not? It seems easier to automate. We could add it to `make fix` and have a validator in CI.

Reply:
Mostly because I want to be very deliberate in choosing the names for these. For example, many of the packages which are primarily used with LZ roll up their usage to LZ: https://github.com/GoogleCloudPlatform/blueprints/blob/main/catalog/log-export/folder/pubsub-export/export.yaml#L22

We should have a CI check to confirm that labels are present though.

(For version updates, we will use [release-please](https://github.com/googleapis/release-please).)